packages/lit-html/src/test/test-utils/memory.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright 2017 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+declare global {
+  interface Window {
+    gc: () => void;
+  }
+  interface Performance {
+    memory: {
+      usedJSHeapSize: number;
+    };
+  }
+}
+
+const canRunMemoryTests =
+  globalThis.performance?.memory?.usedJSHeapSize && window.gc;
+
+export const memorySuite = canRunMemoryTests ? suite : suite.skip;
+
+export const forceGC = () => {
```

Comment:
Still wan to add a comment about why this is being called 10x.

Reply:
This file actually goes with the next PR. Will test there and remove if unneeded.